### ISSUE TITLE
Double is_muted

### ISSUE THREAD

**Issue by rysson** [feluxe/sty#18]
Could explain me why `is_muted` is declared twice? 
First time in class Base (as class property), second time in object dict?

```python
from sty import ef
print(f'class mute: {type(ef).is_muted}, object mute {ef.get("is_muted")}')
mute(ef)
print(f'class mute: {type(ef).is_muted}, object mute {ef.get("is_muted")}')
```
```
class mute: False, object mute None
class mute: False, object mute True
```

**Comment by feluxe** [comment]
Hmm, I guess this is a left over from the last version. It seems like it's useless since `1.0.0b9`.

I think I can safely remove ` cls.is_muted: bool = False` from the Base class:

```python

    def __new__(cls):

        cls.is_muted: bool = False
        cls.renderfuncs: Dict[Render, Callable] = {}
        cls.styles: Dict[str, Tuple[Render, ...]] = {}

        return super(Base, cls).__new__(cls)
```

All tests pass without it. I think I'll remove it in the next release.

Thanks for pointing this out! :)

**Comment by rysson** [comment]
You welcome.
Cleanup is a good idea.

**Comment by feluxe** [comment]
Fixed with 825c07b39136c49a3eea9a37e941cc05156faa57
